hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.module.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../../assets/abstracts/mixins";
+@import "../../../assets/abstracts/variables";
+
+.modal {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.modalGrid {
+    background-color: white;
+    padding: 1em;
+    @include responsive(sm-down) {
+        width: 200px;
+    }
+    width: 350px;
+}
+
+.title {
+    text-align: center;
```

Comment:
add font-size:25px here, the extremely large title just didn't look right to me 😅

Reply:
alright